publisher_scaled_joint_trajectory_controller:
  ros__parameters:

    controller_name: "scaled_joint_trajectory_controller"
    wait_sec_between_publish: 6

    goal_names: ["pos1"]
    pos1:
      positions: [-1.57, -1.57, 1.57, 1.57, 1.57, 3.1416]

    joints:
      - prefix_test_shoulder_pan_joint
      - prefix_test_shoulder_lift_joint
      - prefix_test_elbow_joint
      - prefix_test_wrist_1_joint
      - prefix_test_wrist_2_joint
      - prefix_test_wrist_3_joint

    check_starting_point: false
    starting_point_limits:
      shoulder_pan_joint: [-0.1,0.1]
      shoulder_lift_joint: [-1.6,-1.5]
      elbow_joint: [-0.1,0.1]
      wrist_1_joint: [-1.6,-1.5]
      wrist_2_joint: [-0.1,0.1]
      wrist_3_joint: [-0.1,0.1]

publisher_joint_trajectory_controller:
  ros__parameters:

    controller_name: "joint_trajectory_controller"
    wait_sec_between_publish: 6

    goal_names: ["pos1"]
    pos1:
      positions: [-1.57, -1.57, 1.57, 1.57, 1.57, 3.1416]

    joints:
      - prefix_test_shoulder_pan_joint
      - prefix_test_shoulder_lift_joint
      - prefix_test_elbow_joint
      - prefix_test_wrist_1_joint
      - prefix_test_wrist_2_joint
      - prefix_test_wrist_3_joint

    check_starting_point: false
    starting_point_limits:
      shoulder_pan_joint: [-0.1,0.1]
      shoulder_lift_joint: [-1.6,-1.5]
      elbow_joint: [-0.1,0.1]
      wrist_1_joint: [-1.6,-1.5]
      wrist_2_joint: [-0.1,0.1]
      wrist_3_joint: [-0.1,0.1]
